src/DictVectors/abstractdvec.jl

```diff
@@ -252,12 +252,11 @@ Compute the number of walkers in `v`. It is used for updating the shift. Overloa
 function for modifying population control.
 
 In most cases `walkernumber(v)` is identical to `norm(v, 1)`. For `AbstractDVec`s with
-complex coefficients it reports the one norm separately for the real and the imaginary part
+complex integer coefficients it reports the one norm separately for the real and the imaginary part
 as a `ComplexF64`. See [`Norm1ProjectorPPop`](@ref).
 """
-walkernumber(v) = walkernumber(StochasticStyle(v), v)
-# use StochasticStyle trait for dispatch
-walkernumber(::StochasticStyle, v) = dot(Norm1ProjectorPPop(), v)
+walkernumber(v) = norm(v, 1)
+walkernumber(v::AbstractDVec{<:Any,V}) where {I<:Integer,V<:Complex{I}} = dot(Norm1ProjectorPPop(), v)
```

Comment:
Since you don't need `I` and `V`, you can do it like this:
```suggestion
walkernumber(v::AbstractDVec{<:Any,<:Complex{<:Integer}}) = dot(Norm1ProjectorPPop(), v)
```